manifests/profile/vim.pp

```diff
@@ -15,4 +15,11 @@
     content => template('nebula/profile/vim/vimrc.vim.erb'),
     require => Package['vim'],
   }
+
+  # Write an empty /root/.vimrc to prevent vim from automatically
+  # loading /usr/share/vim/vim*/defaults.vim
+  file { '/root/.vimrc':
+    ensure => 'file',
+    mode   => '0644',
+  }
```

Comment:
Hoping this is valid code. I just want to touch the file if it's missing. IDK what's in the file.

Reply:
If it doesn't, you can add `content => ''`

Reply:
Alternatively,

```puppet
exec { '/usr/bin/touch /root/.vimrc':
  creates => '/root/.vimrc',
}```

Reply:
Tested this on two different hosts, does exactly what I wanted: creates the file, but does not change/delete the content if it exists.